html/mycakeapp/src/Controller/AuctionController.php

```diff
@@ -197,6 +197,9 @@ public function shipping($bidinfo_id = null)
 		$this->set('bidinfo', $bidinfo);
 		$biditem = $this->Biditems->get($bidinfo['biditem_id']);
 		$this->set('biditem', $biditem);
+
+		$review = $this->Reviews->find('all')->where(['reviewed_id' => $bidinfo['user_id']])->where(['reviewed_id' => $biditem['user_id']])->toArray();
```

Comment:
(修正不要)横に長いコードは改行すると読みやすいですね！
```suggestion
		$review = $this->Reviews->find('all')
			->where(['reviewed_id' => $bidinfo['user_id']])
			->where(['reviewed_id' => $biditem['user_id']])
			->toArray();
```